build/AppxManifest.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
+  xmlns:desktop="http://schemas.microsoft.com/appx/manifest/desktop/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities">
+  <Identity Name="Deltares.Ribasim" Version="2025.6.0.0"
```

Comment:
The `Identity Name="Deltares.Ribasim"` part is important to make sure we don't have to change that later on.
I'm not aware of other MSIX packages at Deltares (none in the Microsoft Store). But it could be that e.g. people want us to use "Stichting Deltares" rather than "Deltares". If we have to change that, I think they essentially become two separate Ribasims, which we want to avoid.